projects/packages/jetpack-mu-wpcom/src/features/verbum-comments/src/components/logged-out.tsx

```diff
@@ -11,8 +12,12 @@ interface LoggedOutProps {
 }
 
 const getLoginCommentText = () => {
-	const defaultText = translate( 'Log in to leave a reply.' );
-	const optionalText = translate( 'Leave a reply. (log in optional)' );
+	const defaultText = commentParent.value
+		? translate( 'Log in to leave a reply.' )
+		: translate( 'Log in to leave a comment.' );
+	const optionalText = commentParent.value
+		? translate( 'Leave a reply. (log in optional)' )
+		: translate( 'Leave a comment. (log in optional)' );
 	const nameAndEmailRequired = translate(
 		'Log in or provide your name and email to leave a reply.'
```

Comment:
We should update this text as well:
`Log in or provide your name and email to leave a reply.`
`Log in or provide your name and email to leave a comment.`

WDYT?

Reply:
Good catch. Works for me!